frontend/public/locales/zh/translation.json

```diff
@@ -400,7 +400,7 @@
   "The meshConfig.meshMTLS.minProtocolVersion field specifies the minimum TLS version for the TLS                   connections among Istio workloads. N/A if it was not set.": "The meshConfig.meshMTLS.minProtocolVersion field specifies the minimum TLS version for the TLS                   connections among Istio workloads. N/A if it was not set.",
   "The Side Panel shows details about the currently selected node or edge, otherwise the whole graph.": "侧面板显示有关当前选定节点或边的详细信息，否则显示整个视图。",
   "The Side Panel shows details about the currently selected node or edge, otherwise the whole mesh.": "The Side Panel shows details about the currently selected node or edge, otherwise the whole mesh.",
-  "There are not Workloads with sidecar for this service": "此服务没有具有sidecar的工作负载",
+  "There are no Workloads with sidecar for this service": "There are no Workloads with sidecar for this service",
```

Comment:
I would leave the Chinese translation alone. We would rather have a true translation that might have slightly wrong syntax than only English/no translation at all. And it actually looks like the translation is correct anyway. Look at [Google Translate](https://translate.google.com/?sl=auto&tl=en&text=%E6%AD%A4%E6%9C%8D%E5%8A%A1%E6%B2%A1%E6%9C%89%E5%85%B7%E6%9C%89sidecar%E7%9A%84%E5%B7%A5%E4%BD%9C%E8%B4%9F%E8%BD%BD&op=translate):

![image](https://github.com/user-attachments/assets/0db3e667-0186-4624-9921-1960bdb4b768)


```suggestion
  "There are no Workloads with sidecar for this service": "此服务没有具有sidecar的工作负载",
```